<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDN connection tester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabProducts">
       <attribute name="title">
        <string>Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonUpdateProducts">
          <property name="text">
           <string>Update from NDN</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableView" name="tableViewProducts">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonGetProducts">
          <property name="text">
           <string>Get data from NDN</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLabel" name="labelRecordsCount">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonProductFilterOptions">
          <property name="text">
           <string>Filter options</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProductGroups">
       <attribute name="title">
        <string>Product groups</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableView" name="tableViewProductGroups">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonGetProductGroups">
          <property name="text">
           <string>Get NDN data</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabManufacturers">
       <attribute name="title">
        <string>Manufacturers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonGetManufacturers">
          <property name="text">
           <string>Get data from NDN</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableView" name="tableViewManufacturers">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVatRates">
       <attribute name="title">
        <string>VAT Rates</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonGetVATRates">
          <property name="text">
           <string>Get from NDN</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonUpdateVATRates">
          <property name="text">
           <string>Update from NDN</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="tableViewVATRates">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionNDN_connection"/>
   </widget>
   <widget class="QMenu" name="menuMonitoring">
    <property name="title">
     <string>Transactions</string>
    </property>
    <addaction name="actionNDN_logs"/>
    <addaction name="actionTransaction_tester"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuMonitoring"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNDN_connection">
   <property name="text">
    <string>NDN connection</string>
   </property>
  </action>
  <action name="actionNDN_logs">
   <property name="text">
    <string>Monitoring</string>
   </property>
  </action>
  <action name="actionTransaction_tester">
   <property name="text">
    <string>Transaction tester</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
